autoload/neomake/makers/ft/vim.vim

```diff
@@ -14,8 +14,11 @@ function! neomake#makers#ft#vim#vint() abort
 
     return {
         \ 'args': l:args,
-        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m'
-        \ }
+        \ 'errorformat': '%I%f:%l:%c:style_problem:%m,%f:%l:%c:%t%*[^:]:%m',
+        \ 'postprocess': {
+        \   'fn': function('neomake#postprocess#GenericLengthPostprocess'),
+        \   'pattern': '\vUndefined variable: \zs([^ ]+)',
```

Comment:
I haven't explored this much, but would this be easier to maintain if it was also allowed to be a list of patterns?